static/themes.ts

```diff
@@ -26,15 +26,15 @@ import $ from 'jquery';
 import {editor} from 'monaco-editor';
 import {SiteSettings} from './settings';
 
-export type Themes = 'default' | 'dark' | 'darkplus';
+export type Themes = 'default' | 'dark' | 'darkplus' | 'system';
```

Comment:
I guess we're stuck with `default` being "light" now. We ca'nt rename that without some trickery, right? Mayybe worth making it "Light" and then having a one-time fixup when we load from the settings `if default return lght` stuff

Reply:
I think the key might be stuck to default, but we could use a different displayname